flow-typed/Claim.js

```diff
@@ -66,6 +66,7 @@ declare type CollectionId = ClaimId | string; // uuid
 
 declare type ChannelId = ClaimId;
```

Comment:
:small_red_triangle: Error when editing a publish claim

I did a bunch of stuff, so not sure how to reproduce.  Here's what I can remember:
- Public list had 4 items
- I added 2
- When publishing, I sorted them a bit, added a few tags, and changed the language to Nederlands.

<img width="1276" alt="image" src="https://user-images.githubusercontent.com/64950861/191255240-9f5b8f1b-a04d-4727-bd83-a847d74386fe.png">

Reply:
I think it's the tags?

<img width="252" alt="image" src="https://user-images.githubusercontent.com/64950861/191256955-7ef06960-d9a2-4708-b860-43d90d0ce863.png">